wren-launcher/utils/rc.go

```diff
@@ -0,0 +1,169 @@
+// the wrenrc.go package is responsible for writing and reading rcfile
```

Comment:
@onlyjackfrost you can use the functions provided by this file to write/read rcfile.